flink-state-backends/flink-statebackend-heap-spillable/pom.xml

```diff
@@ -37,5 +37,20 @@ under the License.
 	<packaging>jar</packaging>
 
 	<dependencies>
+		<!-- core dependencies -->
+
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-streaming-java_${scala.binary.version}</artifactId>
+			<version>${project.version}</version>
+			<scope>provided</scope>
+		</dependency>
+
+                <!-- test dependencies -->
```

Comment:
Tabs / spaces

Reply:
The changes on pom.xml are no more needed with PR #9501 in